README.md

```diff
@@ -112,13 +112,13 @@ import (
 )
 
 func main() {
-   sdk.Run(&sdk.NewProcessorFunc(
-      sdk.Specification{Name: "simple-processor"}),
+   sdk.Run(sdk.NewProcessorFunc(
+      sdk.Specification{Name: "simple-processor", Version: "v1.0.0"},
```

Comment:
@lovromazgon WDYT about `NewProcessorFunc` verifying that the spec contains a name and a version?

Reply:
Maybe we could set default values if those are not provided? Or maybe just version. That way the processor would always pick the latest when not provided.

Reply:
Hmm, yeah maybe we can fill out the version in `NewProcessorFunc` in case it's not provided.

Reply:
https://github.com/ConduitIO/conduit-processor-sdk/pull/31

Reply:
Given that PR and given https://github.com/ConduitIO/conduit/pull/1439 we should be able to remove the version here @simonl2002.

Reply:
Version removed